Main.java

```diff
@@ -9,7 +11,26 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // Create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    // Asks the user to input a number
+    System.out.println("Please enter an integer:");
+
+    // gets the number from the user
+    int num1 = input.nextInt();
+
+    //Asks the user to input another number
+    System.out.println("Please enter another integer:");
+
+    // gets another number from the user
+    int num2 = input.nextInt();
+
+    if (num1 > num2){
+      System.out.println("The biggest number is " + num1);
+      } else {
+      System.out.println("The biggest number is " + num2);
+    }

```

Comment:
Make sure that you are adding comments to your code. Think of the steps your program makes and turn these into comments. Also, watch your indentation. If there is code inside of an if statement, it should be TABBED over an extra bit to show it is inside of the if-statement. The else statement though should be back at the same level as the if statement is.

You also seem to be missing the case if they are the same. This was noted in the question itself. Be sure you are carefully reading the specifications. This is most likely why it failed the test case.